src/language/class-modifiers.md

```diff
@@ -47,27 +49,36 @@ entire interface, use the `abstract` modifier.
 Abstract classes cannot be constructed from any library, whether its own or
 an outside library. Abstract classes often have [abstract methods][].
 
+<?code-excerpt "language/lib/class_modifiers/ex1/a.dart"?>
 ```dart
 // Library a.dart
-abstract class Vehicle { 
-  void moveForward();     // Abstract method.
+base class Vehicle {
+  void moveForward(int meters) {
+    //...
+  }
 }
 ```
 
+<?code-excerpt "language/lib/class_modifiers/ex1/b.dart"?>
 ```dart
+
 // Library b.dart
 import 'a.dart';
 
-var myCar = Vehicle();       // Error: Cannot be constructed
+var myCar = Vehicle(); // Error: Cannot be constructed
 
-class Car extends Vehicle {  // Can be extended
-    int passengers;
-    // ...
+base class Car extends Vehicle {
+  // Can be extended
```

Comment:
Looking at this and the other "Can be" and "Cannot be" comments below, I'm a bit worried having them in the class declaration makes it look like it applies to the properties. What do you think about putting them above instead?